data/plugins/linewrapping.lua

```diff
@@ -23,6 +23,8 @@ config.plugins.linewrapping = common.merge({
   enable_by_default = false,
   -- Requires tokenization
   require_tokenization = false,
+  -- List of Lua patterns matching files extensions to linewrap.
+  files = { "%.txt$", "%.md$", "%.markdown$" },
```

Comment:
Should this be applied for new unnamed files too?

Reply:
I don't think it should, or at least not until the file is named and saved.